pkg/decode/decode.go

```diff
@@ -18,7 +18,7 @@ func MasterConfig(content []byte) (*legacyconfigv1.MasterConfig, error) {
 
 	serializer := k8sjson.NewYAMLSerializer(k8sjson.DefaultMetaFactory, scheme.Scheme, scheme.Scheme)
 	if _, _, err := serializer.Decode(content, nil, masterConfig); err != nil {
-		return nil, errors.Wrap(err, "Failed to decode master config, see error")
+		return nil, errors.Wrap(err, "Failed to decode master config. Check ssh user permissions and file content, see error")
```

Comment:
For this one and following `serializer.Decode` calls:
We should be able to detect a ssh error earlier -> connection/permission/etc/ issues and go that far in such cases.
So "Check file content" should suffice.